recipes/getorganelle/meta.yaml

```diff
@@ -10,20 +10,20 @@ source:
   sha256: d9f8b0e482d62e79fb2fed76e9b1c1491674ce3d5d2828b0bda508c1bf5b7094
 
 build:
-  number: 0
+  number: 1
   script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
   noarch: python
 
 requirements:
   host:
-    - python >3
+    - python
     - pip
     - blast >=2.3
-    - spades >=3.9
+    - spades <=3.13.0
     - bowtie2 >=2.3
 
   run:
-    - python >3
+    - python
     - perl
     - pigz
     - spades >=3.9
```

Comment:
You can probably remove spades (and blast and bowtie2) from the host: section. Does it need spades between 3.9 and 3.13 for running?

Reply:
Currently, it needs spades (and blast and bowtie2) during the installation process, although those requirements will be removed in the coming GetOrganelle version.
Thanks for pointing out the spades version issue. Corrected.